tests/test_browser.py

```diff
@@ -2573,6 +2573,20 @@ def test_html5_webgl_create_context(self):
       print(opts)
       self.btest(path_from_root('tests', 'webgl_create_context.cpp'), args=opts + ['-lGL'], expected='0')
 
+  @requires_graphics_hardware
+  def test_html5_webgl_create_context_ext_no_antialias(self):
+    for opts in [[], ['-O2', '-g1', '--closure', '1'], ['-s', 'FULL_ES2=1']]:
```

Comment:
This testing seems excessive to me given the new function is almost identical to the old. How about just `-O2 --closure 1`?

Also, I don't think we need the antialias variant, just `test_html5_webgl_create_context_ext` is enough.

Reply:
done